src/app/blogs/[slug]/page.tsx

```
@@ -0,0 +1,102 @@
+        <article
+          className="
+            prose prose-invert prose-lg max-w-none
+            bg-dark-lighter
+            backdrop-blur-md
+            rounded-xl
+            p-6 md:p-10
+            shadow-2xl
+            transition-colors duration-300
+            space-y-3
+          "
+          dangerouslySetInnerHTML={{ __html: blog.contentHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)